en/rocky/8/guides/RL_rsnapshot_backup.md

```diff
@@ -0,0 +1,361 @@
+ rsync                             x86_64                       3.1.3-9.el8                          baseos                       404 k
+
+Transaction SummaryIn this case, rsnapshot is going to be run locally to backup a particular machine. For this example,  the configuration file will be broken down for specific things that need to be changed in the file.
+========================================================================================================================================
+Install  3 Packages
+
+Total download size: 543 k
+Installed size: 1.2 M
+Is this ok [y/N]: y
+```
+## Mounting A Drive or Filesystem For Backup
+
+This particular step is only necessary if you are backing up a single machine or server (our first example below).
+
+For this particular step, we are assuming an external hard drive of some sort that will be attached to the USB port on your machine, which would likely be the most common method of backing up a single machine. 
+
+1. Plugin the USB drive. 
+2. Type `dmesg | grep sd` which should show you the drive. Example `EXT4-fs (sda1): mounting ext2 file system using the ext4 subsystem` so in this case, sda1.
+3. Unfortunately (or fortunately depending on your opinion) most modern Linux desktop operating systems automount the drive if they can. We are going to want to change this. To determine this mount point, take the drive information revealed in the dmesg command above and type `mount | grep sda1`, which should show something like this: `/dev/sda1 on /media/username/8ea89e5e-9291-45c1-961d-99c346a2628a`
+4. Type `sudo umount /dev/sda1`
+5. Next, create a mount point for the backup: `sudo mkdir /mnt/backup`
+6. Now mount the drive to this location: `sudo mount /dev/sda1 /mnt/backup`
+7. Now type `mount | grep sda1` again, and you should see something like this: `/dev/sda1 on /mnt/backup type ext2 (rw,relatime)`
+8. Next create a directory that must exist for the backup to continue on the mounted drive. We are using storage for this example: `sudo mkdir /mnt/backup/storage`
+
+Note that for a single machine, you will have to either repeat the umount and mount steps each time the drive is plugged in again or each time the system reboots, or automate these commands with a script.
+
+# Configuring Rsnapshot
+
+This is the most important step. One of the easiest mistakes to make here is to put spaces between things. The rsnapshot configuration requires tabs for any separation between elements. A warning to that effect is at the very top of the configuration file. A space entered somewhere will cause the entire configuration-and your backup-to fail. The best thing is that the default configuration that comes with rsnapshot only needs minor changes to make it work for a backup of a local machine. It's always a good idea, though, to make a backup copy of the configuration file before you start editing:
```

Comment:
"spaces between things" - can we be more specific here? I know you get into more detail later, but it might be helpful here.